Composer/packages/client/src/pages/design/index.tsx

```diff
@@ -279,15 +288,25 @@ function DesignPage(props) {
           }, [] as IBreadcrumbItem[])
         : [];
     return (
-      <Breadcrumb
-        items={items}
-        ariaLabel={formatMessage('Navigation Path')}
-        styles={breadcrumbClass}
-        data-testid="Breadcrumb"
-        onRenderItem={onRenderBreadcrumbItem}
-      />
+      <Fragment>
+        <Breadcrumb
+          items={items}
+          ariaLabel={formatMessage('Navigation Path')}
+          styles={breadcrumbClass}
+          data-testid="Breadcrumb"
+          onRenderItem={onRenderBreadcrumbItem}
+        />
+        <Link
+          style={{ position: 'absolute', right: 0, marginTop: '22px', marginRight: '10px' }}
+          onClick={() => {
+            setDialogJsonVisibility(!dialogJsonVisible);
```

Comment:
The safe way to do this is:
```suggestion
            setDialogJsonVisibility(current => !current);
```